parm/config/gfs/config.atmensanlobs

```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env bash
+
+########## config.atmensanlobs ##########
+# Pre Atm Ens Analysis specific
+
+echo "BEGIN: config.atmensanlobs"
+
+# Get task specific resources
+. "${EXPDIR}/config.resources" atmensanlobs
+
+export JCB_ALGO_YAML=@JCB_ALGO_YAML@
```

Comment:
Do we expect these to need to be over-ridden instead of just set to a static file?

Reply:
`@JCB_ALGO_YAML@` was used to allow developers to specify their own variant of the algorithm yaml via an experiment configuration yaml.

Reply:
I think the question is how exposed to make this setting. All of the stuff in the config files are already user-customizable;  `@_@` just allows these to be set automatically at the (CI) case/host level for repeatability. So far we have been conservative about which variables we give that treatment. I'm not against it (particularly if there is a plan to change it for different CI tests), I just want to square it with our practices so far.

CC @aerorahul

Reply:
Thanks for the explanation.  I was under the impression that we wanted developers to customize by editing yamls which then populate config files.  Of course, developers can always edit config files in their g-w snapshot or in their EXPDIR.

I'm fine with whatever the g-w team wants.

I can retain
```
export JCB_ALGO_YAML=@JCB_ALGO_YAML@
```
in `config.atmensanlobs` and `config.atmensanlsol`, 

Alternatively, I can change `config.atmensanlobs` to
```
export JCB_ALGO_YAML="${PARMgfs}/gdas/atm/jcb-prototype_lgetkf_observer.yaml.j2"
```
and `config.atmensanlsols` to 
```
export JCB_ALGO_YAML="${PARMgfs}/gdas/atm/jcb-prototype_lgetkf_solver.yaml.j2"
```

Let me know what you want and I'll make it so.

Reply:
`@JCB_ALGO_YAML@` is fine.